### PR TITLE
chore: testing the new gcf-utils with canary

### PR DESCRIPTION
**Pull request by tmatsuo**
Now it throws intentionally for an issue titled `canary-bot error` .

### CODE REVIEW

**Review by chingor13 — APPROVED** [review]
Another option is to have an alternative logic branch. Perhaps something like if the title matches `canary-bot error`, it will throw an error.

**Comment by tmatsuo** [comment]
Ah that might be a better idea

**Comment by tmatsuo** [comment]
@chingor13 I added a new code block. PTAL

File: packages/canary-bot/test/test.ts [hunk 1]
```diff
@@ -198,6 +198,16 @@ describe('canary-bot', () => {
       await probot.receive({name: 'issues', payload, id: 'abc123'});
       sinon.assert.notCalled(addOrUpdateIssueCommentStub);
     });
+    it('thorws an error for canary-bot error', async () => {
```
Comment (chingor13) [inline packages/canary-bot/test/test.ts:201]:
```suggestion
    it('throws an error for canary-bot error', async () => {
```

**Review by chingor13 — APPROVED** [review]

**Pull request merged**